Documentation/install/upgrade.rst

```diff
@@ -427,53 +427,79 @@ New ConfigMap Options
-
-    An indicator that this is happening would be drops visible in Hubble or
-    Cilium monitor with a source identity equal to 6 (the numeric value for the
-    new ``remote-node`` identity.   For example:
+    have a default deny rule for ingress connectivity.  This ensures that
+    network-based liveness and readiness probes from kubelet will not be
+    dropped by network policy. Prior to 1.7.x, Cilium achieved this by always
+    allowing ingress host network connectivity from any host in the cluster.
+
+    As this behavioral change is potentially disruptive, several options have
+    been introduced to allow for seamless upgrades and to allow to opt-out of
+    the new behavior and guarantee compatibility with the previous policy
+    behavior:
+
+    ``enable-remote-node-identity=true`` enables the use of the new remote node
+    identity. New deployments provisioned via Helm will automatically enable
+    this new security feature. Existing deployments have to explicitly opt-in
+    by enabling the feature.
+
+    ``transmit-host-src`` and ``allow-host-src`` have been added to control the
+    sending and receiving behavior of the identity used to represent nodes.
+
+    **Instructions for a seamless upgrade:**
+
+    In order to guarantee a seamless upgrade, the following steps must be
+    followed when upgrading from any < 1.7 version to 1.7:
+
+    1. For the initial upgrade, the options ``transmit-host-src: true``,
+       ``allow-host-src: true``, and ``enable-remote-node-identity: false``
+       must be set. This guarantees that:
+       
+       * Existing NetworkPolicy will continue to work.
+       * New 1.7 nodes will continue to transmit using the old host identity to
+         guarantee compatibility with 1.6 nodes during the rolling upgrade
+         process.
+       * New 1.7 nodes will continue allow traffic from the obsolete host
+         identity when received over the network.
+
+    2. A second rolling upgrade must be performed with the option
+       ``transmit-host-src: false``. This will convert all 1.7 nodes to start
+       transmitting with the new remote-node identity.
+
+    3. A third rolling upgrade must be performed with the option
+       ``allow-host-src: false`` set. This will enable the additional security
+       check and start dropping traffic originating from the host identity
+       received over the network. It is important to make this change
+       independent of enabling the transmission to a avoid a mix of nodes with
+       restrictive receive behavior while old nodes are still transmitting with
+       the legacy host identity.
+
+    4. Network policies must be audited to ensure policies no longer allow on the
+       old behavior of the host entity allowing from all nodes in the cluster.
+       After this step has been completed, remote node identity policy can be
+       enabled by setting ``enable-remote-node-identity: true``.
```

Comment:
I wonder whether there is a practical difference to splitting these into separate steps or whether they could be combined together? Until step 4 is executed, from a policy perspective, allows "from host" will still allow traffic from remote nodes.